features-support/step_definitions/features.js

```diff
@@ -261,30 +264,23 @@ module.exports = function () {
   });
 
   this.Given(/^a user is viewing (.*) repository$/, function (repository, callback) {
-    var dotGitPath = './.git';
+    let archiveUrl = "https://gist.github.com/sponte/1fa36cdb67ae2c8ac0fd70ed6e83b2d8/raw/203b45c21921fda6323a175e1bee246c327dcd60/specs.tar.gz";
```

Comment:
Good idea. Is there one test which does pull down from a remote Git repository -- I'd say don't bother but as I recall we do have some of our own code around that, not just the Node-Git stuff.

Would you mind also adding a note about this in the README, so that other people know the tests are reliant on external data? (To be fair they already were since the tests pulled down a specific repository (this one), I had thought about adding a mini Git repo into this repo to use as test data but that seemed a bit weird).